plugins/inputs/tencentcloudcm/products.go

```diff
@@ -0,0 +1,387 @@
+package tencentcloudcm
+
+import (
+	"encoding/json"
+	"fmt"
+
+	cdb "github.com/tencentcloud/tencentcloud-sdk-go/tencentcloud/cdb/v20170320"
+	clb "github.com/tencentcloud/tencentcloud-sdk-go/tencentcloud/clb/v20180317"
+	"github.com/tencentcloud/tencentcloud-sdk-go/tencentcloud/common"
+	"github.com/tencentcloud/tencentcloud-sdk-go/tencentcloud/common/profile"
+	cvm "github.com/tencentcloud/tencentcloud-sdk-go/tencentcloud/cvm/v20170312"
+	dc "github.com/tencentcloud/tencentcloud-sdk-go/tencentcloud/dc/v20180410"
+	es "github.com/tencentcloud/tencentcloud-sdk-go/tencentcloud/es/v20180416"
+	redis "github.com/tencentcloud/tencentcloud-sdk-go/tencentcloud/redis/v20180412"
```

Comment:
You don't need to define aliases I think, do you?

Reply:
in some cases two different cloud products might have same version.

`.../foo/v20170320`
`.../bar/v20170320`

Would it be better to keep the aliases to better distinguish them?

Reply:
Oh I thought golang will notice that the last part is a version and strips that part, similar to `something/v2`. If it doesn't please keep the aliases.